performant-software/textlab
# Fixed CSS related to split panes, left col in proper place and overflow not showing from contents now

**What this PR Does**
Closes #164
Closes #149
Related: Fixed some CSS issues that were causing the content to overlap the panes on the edges

**Steps to test**
Go here: https://mel-textlab-pr-172.herokuapp.com

Verify that the scrollbars appear in the correct place and that both panel-resizers work as expected

Verify that there are no visual glitches IE issue #149 "passing behind" or editor content overlapping the frame.
